From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sat, 27 Feb 2016 08:40:58 -0700
Subject: fix-flake8-expected-values
Forwarded: not-needed

Upstream will update the expected values when the new version of
python-flake8 hits their testing environment; no need to forward this
trivial patch.
---
 test/flycheck-test.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/test/flycheck-test.el
+++ b/test/flycheck-test.el
@@ -3878,8 +3878,8 @@ Why not:
        :checker python-flake8)
    '(7 1 warning "expected 2 blank lines, found 1" :id "E302"
        :checker python-flake8)
-   '(9 9 info "function name should be lowercase"
-       :checker python-flake8 :id "N802")
+   ;; '(9 9 info "function name should be lowercase"
+   ;;     :checker python-flake8 :id "N802")
    '(12 29 warning "unexpected spaces around keyword / parameter equals"
         :id "E251" :checker python-flake8)
    '(12 31 warning "unexpected spaces around keyword / parameter equals"
